lib/ansible/modules/windows/win_firewall.ps1

```diff
@@ -0,0 +1,55 @@
+#!powershell
+# This file is part of Ansible
+
+# Copyright 2015, Hans-Joachim Kliemeck <git@kliemeck.de>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+
+$result = @{
+    changed = $false
+}
+
+# get params
+$params = Parse-Args $args -supports_check_mode $false
+
+$profile = Get-AnsibleParam -obj $params -name "profile" -type "str" -failifempty $true
```

Comment:
Consider using validateset to make sure the profile is one of the 4 acceptable params (look at Get-AnsibleParam in powershell.ps1)